modules/virt-about-reviewing-resource-consumption.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-about-reviewing-resource-consumption-by-pods.adoc
+//
+
+[id="virt-about-reviewing-resource-consumption-by-pods_{context}"]
+
+= About reviewing resource consumption by pods
```

Comment:
@lmandavi  Im not sure if "by pod" is correct.
But Im not sure what to replace it with since its a mix of by virt-launcher pod or virtual machine

Reply:
@sradco@redhat.com, We can keep it simple: 
About reviewing resource consumption.